Dockerfile.rhel7

```diff
@@ -0,0 +1,25 @@
+FROM openshift/origin-release:golang-1.10 as builder
+
+WORKDIR /go/src/github.com/aerogear/ups-config-operator
+COPY . .
+RUN go build -o ups-config-operator ./cmd/server/main.go
```

Comment:
Is it a requirement to build the application when the image is created?

Reply:
It's not required, but for my purposes it makes things a lot easier.

Note that it's a "multi-stage" image build, so none of the build stuff exists in the final output container -- this building happens in the earlier build stage.

See the cluster monitoring operator from OpenShift, for an example of where this is based on: https://github.com/openshift/cluster-monitoring-operator/blob/master/Dockerfile.rhel7

Reply:
Also note, that this is just an alternative Dockerfile alongside the existing Dockerfile, which remains unchanged for now.